bot.js

```diff
@@ -1,35 +1,59 @@
 var Discord = require('discord.io');
 var logger = require('winston');
-var auth = require('./auth.json');
+var  auth = require('./auth.json');
 
-// Configure logger settings
-logger.remove(logger.transports.Console);
-logger.add(logger.transports.Console, {
+//Configure logger settings
+logger.remove(logger.transports.Console, {
     colorize: true
 });
-
 logger.level = 'debug';
 
-// Initialize Discord Bot
+//Канал, в котором бот может реагировать на сообщения
+const ALLOWED_CHANNEL_ID = "525249065039036426";
+
+
+//Initialize Discord Bot
 var bot = new Discord.Client({
-   token: auth.token,
-   autorun: true,
+    token: auth.token,
+    autorun: true
 });
 
-bot.on('ready', function (evt) {
+bot.on('ready',  function(evt) {
     logger.info('Connected');
     logger.info('Logged in as: ');
     logger.info(bot.username + ' - (' + bot.id + ')');
 });
 
 bot.on('message', function (user, userID, channelID, message, evt) {
-    switch(message) {
-        case 'Ассаляму аляйкум':
-            bot.sendMessage({
-                to: channelID,
-                message: 'Уа аляйкум салям',
-            });
-
-            break;
+    
+    var regexp = /\!thanks\b/gmi;
+
+    //в каком канале пишут?
+    if(channelID !== ALLOWED_CHANNEL_ID) {
```

Comment:
```suggestion
    if (channelID !== ALLOWED_CHANNEL_ID) {
```
1. Formatting
2. Let's move it before `regexp`